<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotifySend</class>
 <widget class="QWidget" name="NotifySend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbColor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>60</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>260</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="backwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <zorder>backwidget</zorder>
  <zorder>lbColor</zorder>
  <zorder>btnClose</zorder>
  <zorder>leText</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
